gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceConfiguration.java

```diff
@@ -109,7 +109,7 @@ public GobblinServiceConfiguration(String serviceName, String serviceId, Config
 
     this.isWarmStandbyEnabled = ConfigUtils.getBoolean(config, ServiceConfigKeys.GOBBLIN_SERVICE_WARM_STANDBY_ENABLED_KEY, false);
     this.isMultiActiveSchedulerEnabled = ConfigUtils.getBoolean(config, ServiceConfigKeys.GOBBLIN_SERVICE_MULTI_ACTIVE_SCHEDULER_ENABLED_KEY, false);
-    this.isMultiActiveExecutionEnabled = ConfigUtils.getBoolean(config, ServiceConfigKeys.GOBBLIN_SERVICE_MULTI_ACTIVE_EXECUTION_ENABLED, false);
+    this.isMultiActiveExecutionEnabled = true;
```

Comment:
temp change for testing. not planning to commit it

Reply:
makes sense... don't forget about this :)